sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/BlobAPITest.groovy

```diff
@@ -368,15 +372,15 @@ class BlobAPITest extends APISpec {
             .blobContentType("type")
             .blobCacheControl(properties.cacheControl())
             .blobContentLanguage(properties.contentLanguage())
-            .blobContentMD5(Base64.getDecoder().decode(properties.contentMD5()))
+            .blobContentMD5(Base64.getEncoder().encode(MessageDigest.getInstance("MD5").digest(defaultData.array())))
```

Comment:
This line of changes is really different than original one. Why change it like this?

Reply:
The previous call was trying to decode the default data that was sent to the service incorrectly. This new call is just constructing the ContentMD5 from the actual source.